src/markdown_generators.js

```diff
@@ -202,9 +239,18 @@ const generateMarkdownRecursive = page => {
     courseData
   )}`
   return {
-    name:     isParent ? path.join(pathToChild, "_index.md") : `${pathToChild}.md`,
+    name:
+      isParent || hasFiles
+        ? path.join(pathToChild, "_index.md")
+        : `${pathToChild}.md`,
     data:     courseSectionMarkdown,
-    children: children.map(generateMarkdownRecursive, this)
+    children: children.map(generateMarkdownRecursive, this),
+    files:    pdfFiles.map(file => {
+      return {
+        name: `${path.join(pathToChild, file["id"].replace(".pdf", ""))}.md`,
+        data: generatePdfMarkdown(file, courseData)
+      }
```

Comment:
when you're returning an object literal like this you can just wrap it in `( )` if you want:

```js
file => ({
  name: ...,
  data: ...
})
```

save two lines :laughing:

Reply:
I think somewhere in the back of my head I knew this.  I'm pretty sure that I started writing the inside of the map by declaring some variables and outputting them with console.log to make sure I was getting the right values.  I consolidated it to one return statement and forgot to take it a step further ;-)